serialization.md

```diff
@@ -26,13 +26,6 @@ This specification uses the following terms:
 	Changing it means creating a new derived image, instead of changing the existing image.
     </dd>
     <dt>
-        Image Filesystem Changeset
-    </dt>
-    <dd>
-        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
-	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
```

Comment:
On Fri, Sep 02, 2016 at 02:48:49PM -0700, Stephen Day wrote:
> > -        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
> > -	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
>
> It is okay to repeat some information to enhance context. Please
> merge these descriptions.

Were you taking about more than just this last line?  I'm fine
including the first (“…files which have been added, changed, or
deleted…”) in the layer entry and have done so with a0ee5c4 → b8ff53a.

I'm still not seeing how repeating “layer-based and union filesystems
can make this efficient!” is useful context, but I can copy/paste that
line in at the end of the layer entry if that's what it takes to get
this merged.  Let me know if it is.